pytest_fixtures/component/virtwho_config.py

```diff
@@ -286,6 +288,24 @@ def deploy_type_cli(
         hypervisor_name, guest_name = deploy_configure_by_script(
             script, form_data_cli['hypervisor-type'], debug=True, org=org_module.label
         )
+    elif deploy_type == "name":
+        command = get_configure_command_option(deploy_type, virtwho_config_cli, org_module.name)
+        hypervisor_name, guest_name = deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], debug=True, org=org_module.label
+        )
+    elif deploy_type == "organization-title":
+        virtwho_config_cli['organization-title'] = org_module.title
+        command = get_configure_command_option(deploy_type, virtwho_config_cli, org_module.name)
+        hypervisor_name, guest_name = deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], debug=True, org=org_module.label
+        )
+    elif deploy_type == "location-id":
+        virtwho_config_cli['location-id'] = default_location.id
+        command = get_configure_command_option(deploy_type, virtwho_config_cli, org_module.name)
+        print(command)
+        hypervisor_name, guest_name = deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], debug=True, org=org_module.label
+        )
```

Comment:
There is a lot of code duplication, you can separate that out in a function and just call that function for multiple conditions !

Reply:
@jyejare Thanks for your suggestion. resolve on the commit https://github.com/SatelliteQE/robottelo/pull/13632/commits/78f472a03ffa50b3b9f524c35f78acd17b26bc0a